packages/request-client.js/src/api/request-network.ts

```diff
@@ -394,11 +394,16 @@ export default class RequestNetwork {
     const copiedRequestParameters = Utils.deepCopy(requestParameters);
     copiedRequestParameters.extensionsData = [];
 
+    const detectionChain =
```

Comment:
FYI I have prepared some refactoring of the payment network creation parameters, upcoming PR.